mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -159,6 +159,9 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "enable_bch_with_tokens" => handle_mmrpc(ctx, request, enable_platform_coin_with_tokens::<BchCoin>).await,
         "enable_slp" => handle_mmrpc(ctx, request, enable_token::<SlpToken>).await,
         "enable_eth_with_tokens" => handle_mmrpc(ctx, request, enable_platform_coin_with_tokens::<EthCoin>).await,
+        "enable_eth_with_non_fungible_tokens" => {
+            handle_mmrpc(ctx, request, enable_platform_coin_with_non_fungible_tokens::<EthCoin>).await
+        },
         "enable_erc20" => handle_mmrpc(ctx, request, enable_token::<EthCoin>).await,
```

Comment:
>  It can also be a struct that contains the URL and any other needed info to be added in the future similar to erc20_tokens_requests: Vec<TokenActivationRequest<Erc20TokenActivationRequest>>,   

Good point!

Reply:
Happy to tell that now implementation provides `enable_nft` support 
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2049#issuecomment-1947785221 with req/res examples